doc/can-observe.md

```diff
@@ -309,6 +309,54 @@ class WidgetViewModel {
 }
 ```
 
+## Extending can-observe with Observables
+
+can-observe recognizes a `can.computedPropertyDefinitions` property; getting or setting a key on the object that matches a key in the `can.computedPropertyDefinitions` object will use this functionality. Specifically, the values in this object are functions which return an observation (typically with [can-observation]). The first time one of these properties is accessed, the function is run, and the observation is cached, to be used for all future use on _that instance_.
+
+### Defining Your Own
+
+We have tried to make it as easy as possible to define your own extensions. To that end, there is a `addComputedPropertyDefinition` helper; it accepts the target object, the property name, and the observation generator function.
+
+> _Note_: this `target` can be an instance (such as the output of `observe({})`) *or* a class (such as `class extends ObserveObject`). If target is a class, it will put this method on the prototype, so all instances get this new behavior.
+
+```javascript
+var helpers = require("can-observe/object/helpers");
```

Comment:
Why would this be `object/helpers`?  I suppose it's for `objects`, but it's not really for `ObserveObject` only (is it?).